realsense2_camera/launch/rs_launch.py

```diff
@@ -96,61 +96,38 @@ def generate_launch_description():
         get_package_share_directory('realsense2_camera'),
         'config'
         )
+
+    realsense_parameters = [set_configurable_parameters(configurable_parameters)]
+    if (LaunchConfiguration('config_file') != ''):
+        realsense_parameters=[set_configurable_parameters(configurable_parameters)
+								             , PythonExpression(["'", config, "/' + ", LaunchConfiguration("config_file")])
+								             ]
+
     if (os.getenv('ROS_DISTRO') == "dashing") or (os.getenv('ROS_DISTRO') == "eloquent"):
         return LaunchDescription(declare_configurable_parameters(configurable_parameters) + [
             # Realsense
             launch_ros.actions.Node(
-                condition=IfCondition(PythonExpression([LaunchConfiguration('config_file'), " == ''"])),
```

Comment:
why removing this?

Reply:
Due to the modification in 
https://github.com/Auterion/realsense-ros/pull/4/files/6fb4b435a085414fe7030abb2fc01803278abbb5#diff-4f3ab596a35ce7da1b0eb2289ffce9ad499d0970f5d8c98a672f6ddbbc99fa5cR99-R105
this is no longer needed, as `realsense_parameters` handles the difference